Packs/Azure-Enrichment-Remediation/Playbooks/playbook-Azure_-_User_Investigation.yml

```diff
@@ -0,0 +1,1378 @@
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      key:
+        simple: AzureUnsuccessSecurityRulesChangeEvents
+      value:
+        complex:
+          root: AzureUnsuccessSecurityRulesChange.tables.rows.[0]
+          accessor: '[1]'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 930,
+          "y": 1120
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "28":
+    id: "28"
+    taskid: 7f8e214d-c0df-4d8b-8867-3fd95e597d66
+    type: regular
+    task:
+      id: 7f8e214d-c0df-4d8b-8867-3fd95e597d66
+      version: -1
+      name: Set anomalies events
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.11/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "22"
+    scriptarguments:
+      key:
+        simple: AzureAnomaliesEvents
+      value:
+        complex:
+          root: AzureAnomalies.tables.rows.[0]
+          accessor: '[1]'
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 30,
+          "y": 760
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: ad023e29-8a0d-4c84-87cf-bec720378b87
+    type: regular
+    task:
+      id: ad023e29-8a0d-4c84-87cf-bec720378b87
+      version: -1
+      name: Set event of logon attempt from uncommon country
+      description: |-
+        Set a value in context under the key you entered. If no value is entered, the script doesn't do anything.
+
+        This automation runs using the default Limited User role, unless you explicitly change the permissions.
+        For more information, see the section about permissions here:
+        https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/6.11/Cortex-XSOAR-Administrator-Guide/Automations
+      scriptName: SetAndHandleEmpty
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "10"
+    scriptarguments:
+      key:
+        simple: AzureUncommonCountryLogonEvents
+      value:
+        complex:
+          root: AzureUncommonCountryLogon.tables.rows.[0]
```

Comment:
Did you choose a specific index on purpose? '[0].[1]'

Reply:
yes.
.[0] it's the events and .[1] it's the number (count) of the events.